core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -219,6 +219,8 @@ def process_directory(jvm_args, study_directory):
     study_meta_dictionary = {}
     cancer_type_filepairs = []
     sample_attr_filepair = None
+    sample_resource_filepair = None
+    resource_defination_filepair = None
```

Comment:
it should be `definition` not `defination` :)

Reply:
I do need a spell checker!!

Reply:
updated!